metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -182,6 +182,7 @@ This package installs the Metron Bundled Parser files
 %{metron_home}/config/zookeeper/parsers/bro.json
 %{metron_home}/config/zookeeper/parsers/snort.json
 %{metron_home}/config/zookeeper/parsers/squid.json
+%{metron_home}/config/zookeeper/parsers/syslog3164.json
```

Comment:
Per the changes introduced in [METRON-1815](https://github.com/apache/metron/pull/1249), the syslog parsers fall under the [metron-parsers-common](https://github.com/apache/metron/tree/master/metron-platform/metron-parsing/metron-parsers-common/src/main/config/zookeeper/parsers) category. 

I could go ahead and modify the spec file to include both syslog 3164 and 5424 under the spec file [here](https://github.com/apache/metron/blob/master/metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec#L140). Let me know.

Reply:
yes, please move the 5424 config from parser to parsers common